# Goal
string map_name
---
# Result
bool success
---
# Feedback (if required, can be omitted)
